scripts/utils/trading_strategy_helpers.js

```diff
@@ -265,30 +263,33 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
         log(
```

Comment:
Initially, I thought it would be smarter to split the buyOrderTransactionCreation and the sellOrderTransactionCreation into two streams of Promises. But then we would no longer be able to write the buy and sell price for one bracket in one line.

Hence I first await Promises, which return [buyOrderPromise, sellOrderPromise]. Then I flat the array and await the inner promises again.

Reply:
I like the approach, it looks good. I wonder what's the difference in gas price at the end. We might even end up saving some gas because we don't need to send the extra data to set the valid from. and I don't see a difference in the number of calls.

Reply:
Yeah, it should be more expensive. But most likely, it is negligible.

Reply:
Maybe more problematic, it should also increase the size of the payload, not by much but we are already close to the limit. I wonder if we should add tests to check that gas cost and payload size stay below a certain limit?